src/main/scripts/src/app/shared/perspectives/collections/post-it/post-it-collections-perspective.component.html

```diff
@@ -1,127 +1,107 @@
+             class="rounded-left text-success"><i class="fa fa-fw fa-search" aria-hidden="true"></i></a>
+          <a *ngIf="postIt.initialized" (click)="onAttributesClick(postIt.collection.code)" title="Attributes"
+             class="rounded-left text-success"><i class="fa fa-fw fa-list-ul" aria-hidden="true"></i></a>
+          <a *ngIf="postIt.initialized" title="Icon picker" class="rounded-left text-success" data-toggle="dropdown"><i
+            class="fa fa-fw fa-eye-dropper" aria-hidden="true"></i></a>
+          <div class="dropdown-menu">
+            <color-picker class="dropdown-item" [(color)]="postIt.collection.color"
+                          (selectedChange)="updateCollection(postIt)"></color-picker>
+            <div class="dropdown-divider"></div>
+            <icon-picker class="dropdown-item" [(icon)]="postIt.collection.icon"
+                         (selectedChange)="updateCollection(postIt)"></icon-picker>
+          </div>
+          <a *ngIf="postIt.initialized && hasManageRole(postIt.collection)"
+             (click)="onPermissionsClick(postIt.collection.code)" title="Permissions"
+             class="rounded-left text-success"><i class="fa fa-fw fa-users" aria-hidden="true"></i></a>
+          <a *ngIf="!postIt.initialized || hasManageRole(postIt.collection)" (click)="confirmDeletion(postIt, modal)"
+             class="text-danger rounded-left"
+             title="Remove Collection"><i class="fa fa-fw fa-times" aria-hidden="true"></i></a>
+        </div>
 
-      <div class="buttons">
-        <a tabindex (click)="onDetailClick(postIt.collection.code)" title="Detail">
-          <i class="fa fa-fx fa-search"></i></a>
-        <a tabindex (click)="onAttributesClick(postIt.collection.code)" title="Attributes">
-          <i class="fa fa-fx fa-list-ul"></i></a>
-        <a tabindex title="Icon picker" (click)="postIt.pickerVisible = !postIt.pickerVisible">
-          <i class="fa fa-fx fa-eye-dropper"></i></a>
-        <a tabindex *ngIf="hasManageRole(postIt.collection)" (click)="onPermissionsClick(postIt.collection.code)"
-           title="Permissions">
-          <i class="fa fa-fx fa-users"></i></a>
-        <a tabindex *ngIf="hasManageRole(postIt.collection)" (click)="onDeleteClick(postIt, modal)" class="remove-icon"
-           title="Remove Collection">
-          <i class="fa fa-fx fa-times" data-fa-transform="right-3"></i></a>
       </div>
-    </ng-template>
 
-    <ng-template #nonInitializedContent>
-      <div class="document-count" title="Document count">0&nbsp;<i class="far fa-file"></i>
-      </div>
+      <div class="card-footer bg-transparent">
+        <div class="row">
+          <textarea class="col" #textArea rows="1" placeholder="Name" [(ngModel)]="postIt.collection.name"
+                    (ngModelChange)="updateToScrollbarHeight(textArea)" (blur)="onTextAreaBlur(postIt, textArea)"
+                    (focus)="textArea.placeholder = ''">
+              </textarea>
```

Comment:
Why have you changed `input` to `textarea`?

Reply:
So that when the user inputs too long name it wraps on the next line instead of getting clipped.